MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionViewQtGUI.h

```diff
@@ -257,15 +257,15 @@ private slots:
   Ui::EnggDiffractionQtTabFocus m_uiTabFocus;
   Ui::EnggDiffractionQtTabPreproc m_uiTabPreproc;
   // Ui::EnggDiffractionQtTabFitting m_uiTabFitting;
-  EnggDiffFittingViewQtWidget *m_fittingWidget;
+  EnggDiffFittingViewQtWidget *m_fittingWidget{nullptr};
   Ui::EnggDiffractionQtTabSettings m_uiTabSettings;
 
   /// converts QList to a vector
   std::vector<std::string> qListToVector(QStringList list,
                                          bool validator) const;
 
   /// instrument selected (ENGIN-X, etc.)
-  std::string m_currentInst;
+  std::string m_currentInst{"ENGINX"};
```

Comment:
I'm not sure I see the need to use member initializers **and** constructor initializer lists here?

We have to write the constructor since the QWidget instance needs to be accepted. Could it not cause confusion if we have the initialization in two places?